packages/shared/src/modules/categories/libs/types/category-get-all-item-response-dto.type.ts

```diff
@@ -1,6 +1,7 @@
-type CategoryGetAllItemResponseDto = {
+type CategoryGetItemResponseDto = {
```

Comment:
let's rename it to how it was, the operation here is getAll and item suffix indicates that it's not a format of a response, but rather an item from it. It's just the convention how we name response/request dto. because getAll usually returns array and to define item from it, we name it getAllItem
```suggestion
type CategoryGetAllItemResponseDto = {
```